### DISCUSSION TITLE
Spektrum GPS

### DISCUSSION THREAD

**Discussion by inventor7777** [pascallanger/DIY-Multiprotocol-TX-Module#993] [Q&A] [answered]
Hi Multi-module team!

I am considering upgrading my plane to the Spektrum 95871 GPS module and the AR637T receiver (AS3X/SAFE) (TX16S II Max)

WIll all functions work as expected? (AS3X/SAFE/GPS/telemetry)? I looked at the protocol information on multi-module.org and it seems to check out, but I saw no mention of whether the GPS/AS3X will work.

**Comment by pascallanger** [comment, marked as answer]
As3f should be fully supported and the configuration can be done through the lua scripts.
For the GPS, I don't know the telemetry is decoded by the radio. So if you run edgetx you can ask them or just try it out.

**Reply by inventor7777** [reply]
Ok! Sounds good, thanks!
